ui/00_message/ui/pq_history.xml

```diff
@@ -6,17 +6,22 @@
   <message>
     <key>pq_history_002</key>
     <original>依頼人</original>
+    <translation>Client</translation>
   </message>
   <message>
     <key>pq_history_003</key>
     <original>使命：%s</original>
+    <translation>Mission: $s</translation>
+    <!--Needs revision, is it ok like this or it would be better to put percentages-->
```

Comment:
When you see something like "%s", it's a a string formatting code. Text will be inserted (and replace) the %s. So you would want to leave it as a %s.